packages/rendering/lib/transform.js

```diff
@@ -28,6 +36,49 @@ BindTransform.prototype.transform = function(ast) {
     var b = this.syntax.builders;
 
     this.syntax.traverse(ast, {
+      ElementNode(node) {
+        var contentProperty,
+            foundDynamicContent = false,
+            newAttributes = [];
+
+        var tag = node.tag;
+        for (var i=0; i < node.attributes.length; i++) {
+          var nodeAttributes = node.attributes[i];
+          var name = nodeAttributes.name;
+          var value = nodeAttributes.value;
+          if (!value) {
+            break;
+          }
+
+          if (value.type === 'MustacheStatement') {
+            var path = value.path;
+            var parts = path.parts;
+            if (parts.length === 2 && parts[0] === 'content') {
+              foundDynamicContent = true;
+              // contentProperty is the property that is looked up on content
+              // (e.g `imageUrl` in the case of `content.imageUrl`)
+              contentProperty = parts[1];
+              newAttributes.push(b.attr(name, b.mustache(b.path('param1'))));
```

Comment:
I think we will get a naming collision if the user happens to be already using something named `param1`.

We can detect this problem statically (by searching through the AST we're wrapping) and pick a free name instead.